manifests/init.pp

```diff
@@ -93,6 +93,8 @@
   $client_body_timeout                                       = '60s',
   $send_timeout                                              = '60s',
   $lingering_timeout                                         = '5s',
+  Optional[String] $lingering_close                          = undef,
```

Comment:
can we enforce the string a bit more? maybe at least `Optional[String[1]]`?

Reply:
done.
for lingering_close list the 3 valid options as enum
for lingering_time enforce a non empty string

Reply:
Good catch @bastelfreak - I totally missed that.